docs/tutorial/force_calibration.rst

```diff
@@ -212,3 +212,102 @@ If we compare the different fits, we can see that the Lorentzian model with diod
+The desired positional calibration is then:
+
+.. math::
+
+    R_d = \sqrt{\frac{W_{physical}}{W_{measured}}} \quad \mathrm{[m/V]}
+
+Note how this time around, we did not rely on assumptions on the viscosity of the medium or the bead size.
+
+As a side effect of this calibration, we actually obtain an experimental estimate of the drag coefficient:
+
+.. math::
+
+    \gamma_{measured} = \frac{k_B T}{R_d^2 D_{measured}} \quad \mathrm{[kg/s]}
+
+How to do active calibration in Pylake
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Using `Pylake`, the procedure to use active calibration is not very different from passive calibration.
+However, it does require some additional data channels as inputs.
+Instead of using the :class:`~.PassiveCalibrationModel` presented in the previous section, we now use the :class:`ActiveCalibrationModel`.
+
+In this tutorial, we're going to assume the nanostage was used as driving input::
+
+    driving_data = f["Nanostage Position"]["Nanostage X"][start:stop]
+
+We also need to provide the sample rate at which the data was acquired, and a rough guess for the driving frequency.
+Pylake will find an accurate estimate of the driving frequency based on this initial estimate (provided that it is close enough)::
+
+    active_model = lk.ActiveCalibrationModel(driving_data.data, volts, driving_data.sample_rate, bead_diameter, driving_frequency_guess=37)
+
+To check the determined frequency, we can look at the determined driving frequency::
+
+    >>> active_model.driving_frequency
+    36.95
+
+We can now use this model to fit the power spectrum::
+
+    calibration = lk.fit_power_spectrum(power_spectrum, active_model)
+    calibration
+
+And that's all there is to it.
+
+Analogously, we can specify `hydrodynamically_correct=True` if we wish to use the hydrodynamically correct theory here.
+This fits the thermal part with the hydrodynamically correct power spectrum and also uses a hydrodynamically correct model for the peak:
+
+.. math::
+
+    P_{hydro, active}(f) = \delta \left(f - f_{drive}\right) \frac{\left(A f_{drive} \left|\gamma / \gamma_0\right|\right)^2}{2 \left(\left(f_{c,0} + f Im(\gamma/\gamma_0) - f^2/f_{m, 0}\right)^2 + \left(f Re(\gamma / \gamma_0)\right)^2\right)}
+
+We can also include a distance to the surface like before.
+This results in the expression for `\gamma` becoming dependent on the distance to the surface.
+This uses the same expression as listed in the section on the :ref:`hydrodynamically correct model<Hydrodynamically correct model>`.
+One thing to note is that when using the hydrodynamically correct model, the equation for the drag _does_ include the viscosity and bead diameter.
+However, they now appear in a term which already amounts to a small correction therefore the impact of any errors in these is reduced :cite:`tolic2006calibration`.
```

Comment:
I might split this into 2 paragraphs with the 2nd paragraph starting at `One thing to note...`. Otherwise, to me, this all just looks like a giant block of text and the note might get overlooked

Reply:
👍